src/ImageAPI/ProvidersImages/Simple/TimedDealProviderImage.cs

```diff
@@ -1,15 +1,16 @@
-﻿using SixLabors.Fonts;
+﻿using ImageAPI.Utils;
+using SixLabors.Fonts;
 using SixLabors.ImageSharp;
+using MetaDataAPI.Providers;
 using MetaDataAPI.Models.DynamoDb;
 
 namespace ImageAPI.ProvidersImages.Simple;
 
 public class TimedDealProviderImage : ProviderImage
 {
-    public override Image Image { get; }
+    public sealed override Image Image { get; }
```

Comment:
why to use `sealed`? better to unseal, and remove some code from the simple providers, got some duplicates there